packages/blockchain-utils-validation/src/blockchains/cosmos.ts

```diff
@@ -0,0 +1,59 @@
+import { AccountID } from "caip";
+import { Tx, SignMeta, verifyTx } from '@tendermint/sig';
+import { BlockchainHandler } from "../blockchain-handler";
+import * as linking from "@ceramicnetwork/blockchain-utils-linking";
```

Comment:
```suggestion
import type { LinkProof } from "@ceramicnetwork/blockchain-utils-linking";
```
You can just import the type here and use it below.